src/core/Msat.ml

```diff
@@ -12,46 +10,57 @@ module type PLUGIN_MCSAT = Solver_intf.PLUGIN_MCSAT
 module type PROOF = Solver_intf.PROOF
 
 (** Empty type *)
-type void = (unit,bool) Solver_intf.gadt_eq
+type void = (unit, bool) Solver_intf.gadt_eq
 
-type lbool = Solver_intf.lbool = L_true | L_false | L_undefined
+type lbool = Solver_intf.lbool =
+  | L_true
+  | L_false
+  | L_undefined
 
-type ('term, 'form, 'value) sat_state = ('term, 'form, 'value) Solver_intf.sat_state = {
+type ('term, 'form, 'value) sat_state =
+      ('term, 'form, 'value) Solver_intf.sat_state = {
   eval : 'form -> bool;
   eval_level : 'form -> bool * int;
   iter_trail : ('form -> unit) -> ('term -> unit) -> unit;
-  model : unit -> ('term * 'value) list;
+  model : unit -> ('term * 'value) list
 }
 
-type ('atom,'clause, 'proof) unsat_state = ('atom,'clause, 'proof) Solver_intf.unsat_state = {
+type ('atom, 'clause, 'proof) unsat_state =
+      ('atom, 'clause, 'proof) Solver_intf.unsat_state = {
   unsat_conflict : unit -> 'clause;
   get_proof : unit -> 'proof;
-  unsat_assumptions: unit -> 'atom list;
+  unsat_assumptions : unit -> 'atom list
 }
+
 type 'clause export = 'clause Solver_intf.export = {
   hyps : 'clause Vec.t;
-  history : 'clause Vec.t;
+  history : 'clause Vec.t
 }
 
-type ('term, 'formula, 'value) assumption = ('term, 'formula, 'value) Solver_intf.assumption =
+type ('term, 'formula, 'value) assumption =
+      ('term, 'formula, 'value) Solver_intf.assumption =
   | Lit of 'formula  (** The given formula is asserted true by the solver *)
-  | Assign of 'term * 'value (** The term is assigned to the value *)
-
-type ('term, 'formula, 'proof) reason = ('term, 'formula, 'proof) Solver_intf.reason =
+  | Assign of 'term * 'value  (** The term is assigned to the value *)
```

Comment:
There should probably be one blank line between the two type defs, but I guess it cna be added after and ocamlformat would keep it ?